packages/openapi-ts/src/utils/write/models.ts

```diff
@@ -133,105 +152,50 @@ const processServiceTypes = (services: Service[]) => {
                     ? baseOrResMap
                     : Array.from(baseOrResMap).map(([code, base]) => {
                           const value: Model = {
-                              $refs: [],
-                              base,
-                              description: null,
-                              enum: [],
-                              enums: [],
-                              export: 'generic',
-                              imports: [],
-                              isDefinition: false,
-                              isNullable: false,
-                              isReadOnly: false,
+                              ...emptyModel,
+                              ...base,
```

Comment:
Does this do anything?

Reply:
Yes, my bad. Base is `Model`. Ive updated to spread base and removed no longer needed fields so the toType can handle all generation of code